### PR TITLE
add shutdown.sh

### PR DESCRIPTION
**Pull request by jgzl**
add shutdown.sh

### CODE REVIEW

File: dist-material/bin/shutdown.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OAP_NAME=org.apache.skywalking.oap.server.starter.OAPServerStartUp
+WEB_NAME=skywalking-webapp
+oapPID=`ps aux|grep $OAP_NAME|grep -v grep|awk '{print $2}'|xargs`
+webPID=`ps aux|grep $WEB_NAME|grep -v grep|awk '{print $2}'|xargs`
```
Comment (wu-sheng) [inline dist-material/bin/shutdown.sh:21]: Typically, the PID should be written by startup shell. Take a Tomcat as an example.

**Comment by wu-sheng** [comment]
Please finish the Windows version too.

File: dist-material/bin/shutdown.sh (hunk 1 above)
Comment (jgzl) [reply dist-material/bin/shutdown.sh:21]: Ok, I'll refer to the stop and start script of Tomcat

**Comment by jgzl** [comment]
> Please finish the Windows version too.

windows版本不太好实现，目前实现的版本需要把相关执行内容直接放到cmd中才可以停止java应用（还不如手动关闭startup.sh来得方便）

**Comment by wu-sheng** [comment]
> > Please finish the Windows version too.
> 
> 
> 
> windows版本不太好实现，目前实现的shutdown.bat版本需要把相关执行内容直接放到cmd中才可以停止java应用（还不如手动关闭startup.bat打开的弹窗来得方便）

Please keep using English on Github. We need all things available to the global.

**Comment by wu-sheng** [comment]
But I still can see your `shutdown.bat`. If we don't need that, please remove it.

The other thing is about `.sh`. We have separated shell scrips for OAP and UI, please make sure you shutdown script matching the pattern.

**Comment by wu-sheng** [comment]
@kezhenxu94 Why this PR shows merge?

**Comment by kezhenxu94** [comment]
> @kezhenxu94 Why this PR shows merge?

I have no idea either, I'm checking, might be a bug of GH, I just merged #7016

**Comment by kezhenxu94** [comment]
> > > @kezhenxu94 Why this PR shows merge?
> > 
> > 
> > I have no idea either, I'm checking, might be a bug of GH, I just merged #7016
> 
> Yes, and this PR seems keeping nothing changed.

OK, I think it's because the author installed the app https://github.com/apps/pull in their fork and when I merge #7016 , the app force pushes to their master branch, which is also the branch this PR uses.

**Comment by dmsolr** [comment]
> @kezhenxu94 Why this PR shows merge?

The same thing happened in [apisix-java-plugin-runner](https://github.com/apache/apisix-java-plugin-runner/pull/9).

**Comment by kezhenxu94** [comment]
> The same thing happened in [apisix-java-plugin-runner](https://github.com/apache/apisix-java-plugin-runner/pull/9).

That PR's status is close, and I can see it's closed by the author, no?

**Comment by dmsolr** [comment]
Because it has been merged. 
image

**Pull request merged**